scripts/transform.py

```diff
@@ -23,30 +25,47 @@ def _transform_bundle(file_path: Path, output_path: Path) -> dict:
     """Read json, update bundle with bundle Specimen, Task, ensure DocumentReference."""
     tic = time.perf_counter()
     bundle = Bundle(json.load(open(file_path)))
-    diagnostic_reports = []
-    document_references = []
+    dna_diagnostic_reports = []
+    dna_document_references = []
+    imaging_diagnostic_reports = []
+    imaging_document_references = []
     patient = None
     additional_entries = []
     conditions = []
+    imaging_studies = []
     for e in bundle.entry:
+
         if e.resource.resource_type == 'Patient':
             patient = e.resource
+
         if e.resource.resource_type == 'DiagnosticReport':
             codes = [c.code for c in e.resource.code.coding]
             # genetic panel
             if '55232-3' in codes:
-                diagnostic_reports.append(e.resource)
+                dna_diagnostic_reports.append(e.resource)
+            # imaging
+            if e.resource.presentedForm and len(e.resource.presentedForm) == 1 and e.resource.presentedForm[0].data:
+                data = base64.b64decode(e.resource.presentedForm[0].data).decode("utf-8")
+                if '.dcm' in data:
+                    imaging_diagnostic_reports.append(e.resource)
+
         if e.resource.resource_type == 'DocumentReference':
             data = base64.b64decode(e.resource.content[0].attachment.data).decode("utf-8")
             if "_dna.csv" in data:
-                document_references.append(e.resource)
+                dna_document_references.append(e.resource)
+            if '.dcm' in data:
+                imaging_document_references.append(e.resource)
+
         if e.resource.resource_type == 'Condition':
             conditions.append(e.resource)
 
+        if e.resource.resource_type == 'ImagingStudy':
+            imaging_studies.append(e.resource)
+
     output_file = None
-    if len(diagnostic_reports) > 0:
-        logging.info(f"{file_path} has {len(diagnostic_reports)} genetic analysis reports")
-        for diagnostic_report in diagnostic_reports:
+    if len(dna_diagnostic_reports) > 0:
```

Comment:
Comments for later work - these aren't blockers to merge this PR:

+ Future checks could include a counter for this and line 118 (length of imaging_diagnostic_reports) to check that all records were captured - none missed to due odd case
+ counts should be 172 patients with dna and imaging (and fhir bundle when added in future PRs), 289 patients lacking imaging or dna file associated with their fhir bundle.